### PR TITLE
Add Recap extension

### PR DESCRIPTION
**Pull request by kvzivn**
## Description

This extension is used to manage your reminders in Recap.

Recap is a simple web application that helps you remember more and retain what matters. 

Add ideas, concepts, arguments, or anything else that you don't want to forget and we will send you simple reminders by email. We use AI to create a summary of whatever you add and use the spaced repetition technique to carefully select the reminders that should be included in each email to maximize long-term retention.

https://getrecap.xyz

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/recap/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Recap
+
+Recap helps you remember more and retain what matters.
```
Comment (pernielsentikaer) [inline extensions/recap/README.md:3]: Could we provide some information about what a recap is and how to obtain it?

Comment (kvzivn) [reply extensions/recap/README.md]: Done!

https://github.com/raycast/extensions/pull/14446/commits/cca3bc26ade60240d041a30d2dbb3f3b9c83b234

**Comment by kvzivn** [comment]
@pernielsentikaer Thank you so much for testing everything so thoroughly!

I completely forgot to implement the check for how many reminders you have left. This is now wired up and published.

What errors did you run into when trying to add the API key into the Raycast extension?

**Comment by pernielsentikaer** [comment]
Seems to work a lot better today, I think I had some network issues 🙂

- Should we either `reset()` after successfully submission OR `popToRoot()`?

**Comment by kvzivn** [comment]
> * Should we either `reset()` after successfully submission OR `popToRoot()`?

@pernielsentikaer Added a reset! Thanks for pointing that out.

**Comment by pernielsentikaer** [comment]
Much better 🙂

✅ Approved

**Pull request merged**